docs/release/role-handbooks/ci-signal/README.md

```diff
@@ -84,3 +85,10 @@ To reduce the amount of flakes please periodically:
 2. Open issues using an appropriate template (flaking-test) for occurring flakes and ideally fix them or find someone who can.
 
    **Note**: Given resource limitations in the Prow cluster it might not be possible to fix all flakes. Let's just try to pragmatically keep the amount of flakes pretty low.
+
+### Post-release cleanup
+Once the new minor release (e.g., `v1.8.0`) has been officially cut, perform the following cleanup:
```

Comment:
Let's also add this to the release tracking issue template
```
* [ ] [CI Manager] [Post-release cleanup](https://github.com/kubernetes-sigs/cluster-api/tree/main/docs/release/role-handbooks/ci-signal#post-release-cleanup)
```
week 18 / l.67 in .github/ISSUE_TEMPLATE/release_tracking.md